.circleci/config.yml

```diff
@@ -52,14 +52,12 @@ workflows:
   test_and_release:
     # Run the test jobs first, then the release only when all the test jobs are successful
     jobs:
-      - test-linux-8
-      - test-linux-10
+      - test-linux-12
```

Comment:
For this to be defined, you should define a `test-linux-12` job as well.

https://github.com/electron/get/blob/c49bf7f30801917c7c59a1be56066e5fc7f69dd6/.circleci/config.yml#L28-L31

Reply:
I'm not very familiar with `circleci`, thanks for the correction